plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml

```diff
@@ -131,13 +133,11 @@
       <project-facet-version facet="com.google.cloud.tools.eclipse.facet" version="1">
           <constraint>
             <and>
-              <or>
-                  <requires facet="jst.web" version="2.2"/>
-                  <requires facet="jst.web" version="2.3"/>
-                  <requires facet="jst.web" version="2.4"/>
-                  <requires facet="jst.web" version="2.5"/>
-              </or>
-              <requires facet="java" version="1.7]"/>
+               <requires
+                     facet="jst.web"
+                     version="[2.2-2.5]">
+               </requires>
+              <requires facet="java" version="[1.4-1.7]"/>
```

Comment:
You're right, but if we pin it to 1.7 then users get errors if they try specifying anything else.  We should be able to run code compiled against 1.3 too, I suppose.